src/routes/dashboard/(admin)/drafts/+page.svelte

```diff
@@ -12,6 +14,27 @@
   const { draft, labs, records, available, selected } = $derived(data);
 </script>
 
+{#if draft !== null}
+  <div class="flex flex-row gap-2">
+    <a
+      href="/dashboard/drafts/{draft.id}/students.csv"
+      class="not-prose preset-filled-primary-500 btn"
+      download
+    >
+      <span><Icon src={ArrowUpTray} class="h-8" /></span>
+      <span>Export student ranks</span>
```

Comment:
Just switching to Title Case for consistency.
```suggestion
      <span>Export Student Ranks</span>
```